logproxy-filter-replace/README.md

```diff
@@ -0,0 +1,33 @@
+# logproxy-filter-replace
+Example filter that replaces text in messages based on pattern. Configure the pattern by
+passing it via the `FILTER_REGEXP` environment variable and the string to replace by passing it via the `REPLACE_STRING` variable.
+
+This plugin can be useful for obfuscating sensitive information in the log messages.
+
+# building
+```
+docker build -t logproxy-filter-replace .
+```
+# deployment
+Below is an example `manifest.yml` for deployment to Cloud foundry:
+
+```
+---
+applications:
+- name: logproxy-filter-obfuscate
+  docker:
+    image: jdelucaa/logproxy-filter-replace:latest
```

Comment:
I haven't published this docker image yet, do you think it would make more sense to publish it to your docker hub account instead of mine?

Reply:
I think it would be a good example to share as part of the core documentation, so happy to merge it in the logproxy-plugins repo if you are OK with that 👍🏻